### PR TITLE
Fix 39mag.benesse.ne.jp - Ad

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

1 - https://39mag.benesse.ne.jp/lifestyle/content/?id=158977
`##li[id^="div-M_MD_gtx_fluid_"]`

2 - https://39mag.benesse.ne.jp/
`#?#.newPostWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)`

3 - https://39mag.benesse.ne.jp/tags/?id=278
`#?#.postListWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)`

### Add your comment and screenshots

<details><summary>screenshot - 1</summary>

</details><br/>

<details><summary>screenshot - 2</summary>

</details><br/>

<details><summary>screenshot - 3</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -552,7 +552,8 @@ katuru.com#?#.ui-block-b:has(script:not([src]))
 warotanikki.com#?#.widget_diver_widget_pcsp > div > div.adp:upward(2)
 warotanikki.com#?#.widget_diver_widget_pcsp > div > center > script[src$="/adstir.js"]:upward(3)
 blogterest.net#?#.blockSide > div.blockWrapper > div > div[class$="eroterest_partsv2"]:upward(2)
-39.benesse.ne.jp#?#.newPostWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
+39mag.benesse.ne.jp,39.benesse.ne.jp#?#.newPostWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
+39mag.benesse.ne.jp#?#.postListWrap_item > div[id^="div-M_MD_gtx_"]:upward(1)
```
Comment (Yuki2718) [inline JapaneseFilter/sections/css_extended.txt:556]: Unite them to single rule `#?#li[class$="_item"] > div[id^="div-M_MD_gtx_"]:upward(1)`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by Yuki2718** [comment]
Add `39mag.benesse.ne.jp##.recommend` too to JPF and
```
39mag.benesse.ne.jp##.ad-inarticle-banner
39mag.benesse.ne.jp##div[style="min-height:600px;"]
```
to Mobile.

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/css_extended.txt:556]: OK!

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -4606,8 +4606,10 @@ kobe-np.co.jp##.prNews
 kobe-np.co.jp##div[id^="knpc-"]
 st.benesse.ne.jp##div[id^="div-M_MD_kiji_fluid_"]
 benesse.ne.jp##div[style^="height: 250px"]
+39mag.benesse.ne.jp##.recommend
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:4609]: We have `estate.nikkan.co.jp,marieclairejapon.com,daily.co.jp,post.tv-asahi.co.jp,milk-key.com,fujinkoron.jp##.recommend`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:4609]: sorry

**Review by Yuki2718 — APPROVED** [review]

**Pull request closed without merging**